_docker/environments/drupal-production/docker-compose.yml

```diff
@@ -43,9 +43,9 @@ services:
       - "80:80"
     volumes:
       - /credentials/drupal/rhd.settings.yml:/var/www/drupal/web/sites/default/rhd.settings.yml
-      - ./rhd.settings.php:/var/www/drupal/web/sites/default/rhd.settings.php
-      - ../../../images:/var/www/html/images:ro
-      - ../../../stylesheets/fonts:/var/www/html/fonts:ro
+      - /credentials/drupal/rhd.settings.php:/var/www/drupal/web/sites/default/rhd.settings.php
+      - ../../../images:/var/www/drupal/web/images:ro
+      - ../../../stylesheets/fonts:/var/www/drupal/web/fonts:ro
```

Comment:
We need to ensure that we have the bind-mount for the `/var/www/drupal/web/sites/default/images` directory so that images uploaded to Drupal are stored on the NAS disk attached to the VM.

Perhaps something like this to make it extra clear what purpose the bind mount serves?

```
volumes:
 - /data/drupal-images:/var/www/drupal/web/sites/default/images
```

What do you think?